server/notifications/message/plain.go

```diff
@@ -0,0 +1,29 @@
+package message
+
+import (
+	"fmt"
+
+	"github.com/Hukyl/genesis-kma-school-entry/models"
+)
+
+// PlainRateMessage is a message that contains the exchange rate between two currencies.
+type PlainRateMessage struct {
```

Comment:
please, try to avoid repetitive names. https://google.github.io/styleguide/go/decisions#repetition

this type is exported, this means you'll address it from other package like **message**.PlainRate**Message**. Do you see? Message word is duplicated.

I propose to simplify naming to `PlainRate`, this way you'll address this type from another package like `message.PlainText` which is idiomatic. But it's up to you